_posts/2017-06-02-jslang-meetup-async-await.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "Async Function Kata at #jslang Meetup"
+date:   2017-06-02 08:03:00 +0200
+categories: meetup javascript kata
+---
+
+The callback-hell times are long over. We have better tools now. But which of the new kids on the block do you want to 
+play with? In this meetup, last night we tried out [async functions][async-funcs], also known by the name "async/await". 
```

Comment:
I usually try to avoid to refer to something by "last night" because it most certainly is not true or accurate to anyone who is reading it later. Maybe just write something like "In our meetup we tried out ..." ? Just a suggestions.